SteamKit2/SteamKit2/Steam/CMClient.cs

```diff
@@ -142,10 +142,12 @@ public CMClient( SteamConfiguration configuration, string identifier )
 
             ID = identifier;
 
-            heartBeatFunc = new ScheduledFunction( () =>
-            {
-                Send( new ClientMsgProtobuf<CMsgClientHeartBeat>( EMsg.ClientHeartBeat ) );
-            } );
+            heartBeatFunc = new ScheduledFunction(
```

Comment:
Maybe this can be replaced with PeriodicTimer while we're at it

Reply:
I was hoping to not have to do any significant rewriting in this PR.